sicario.c

```diff
@@ -129,6 +178,33 @@ void updateUserMovement(int* xVariation, int* yVariation) {
 	}
 }
 
+bool isColidingWithBoard(int x, int y){
+	return gameBoard[x][y]=='#';
+}
+
+void updateBotMovement(int x, int y, int* xVariation, int* yVariation) {//MELHORAR A IA AQUI
+	if (!chance(100)) {
+		*xVariation = 0;
+		*yVariation = 0;
+	} else if (chance(5)){
+		*xVariation = -1;
+		*yVariation = 0;
+	} else if (chance(5)) {
+		*xVariation = 1;
+		*yVariation = 0;
+	} else if (chance(5)){
+		*xVariation = 0;
+		*yVariation = 1;
+	} else {
+		*xVariation = 0;
+		*yVariation = -1;
+	} 
+	if (isColidingWithBoard(x+*xVariation,y+*yVariation)){
```

Comment:
Why these if is here? This function just update the bot movement, it should not have any bot collision notion.

Reply:
Actually, I disagree. 

This is where the bot would think about where it wanna go. And certainly if where it desires to go is a wall (or another player) it's in his best interest not to do that.

I'm open to suggestions, though.

Reply:
Then, the name of this function should not be updateBotMovement, something like moveBot (by the fact that it takes the movement decision for the bot) is more representative.

Reply:
Indeed, makes sense.